src/vhost_user/master.rs

```diff
@@ -356,12 +356,11 @@ impl VhostUserMaster for Master {
             return error_code(VhostUserError::InvalidOperation);
         }
 
-        // vhost-user spec states that:
+        // TODO: vhost-user spec states that:
```

Comment:
Please remove the `TODO`.

Reply:
I think this addition of "TODO" is due to a revert commit, because I want to keep a clean port:

commit a94720fc4b5d198db684c9e27cc7f221a23583bd
Author: Eryu Guan <eguan@linux.alibaba.com>
Date:   Mon Feb 24 16:04:11 2020 +0800

    Revert "master: enable config messge for get_config()"
    
    This reverts commit 8aedf7685225909253a5887228107ff0ed5d68ae.
    
    The GET_CONFIG command is fixed by commit 927148dd3cfc ("vhost_rs: Fix
    GET_CONFIG command") from cloud-hypervisor in a similar way. Let's take
    that commit and be synced with cloud-hypervisor.

And the TODO got deleted by a later patch:

commit 29fe7579670370f04a656c4f623141bf620b16a5
Author: Sebastien Boeuf <sebastien.boeuf@intel.com>
Date:   Tue Sep 10 15:29:53 2019 -0700

    vhost-user: Fix GET_CONFIG command
    
    Following the vhost-user specification that can be found at
    https://github.com/qemu/qemu/blob/master/docs/interop/vhost-user.rst#virtio-device-config-space
    this patch fixes the GET_CONFIG command.
    
    This command is very specific as it needs to provide a body and a
    payload corresponding to the content of the configuration structure as
    an array of bytes. As a reply, it receives the body and payload provided
    by the slave.
    
    Signed-off-by: Sebastien Boeuf <sebastien.boeuf@intel.com>